README.md

```diff
@@ -1003,6 +1004,15 @@ The following environment variables control the custom response feature:
 1. `LIMIT_REMAINING_HEADER` - The default value is "RateLimit-Remaining", setting the environment variable will specify an alternative header name
 1. `LIMIT_RESET_HEADER` - The default value is "RateLimit-Reset", setting the environment variable will specify an alternative header name
 
+## Optional custom headers (mimicking Github API behavior)
```

Comment:
If we move forward with this; the way the configuration is currently implemented feels a little awkward. In particular you always end up with both versions of the "reset header": 

```
RateLimit-Reset: 58
X-RateLimit-Reset: 1697219026
```

Would it make more sense to have a configuration option like `LIMIT_RESET_HEADER_TIMESTAMP_MODE` that defaults to `false` but when enabled; it writes the timestamp value to the header?

There would still be the potential awkwardness of the `RateLimit-Reset` header having a value that is not in accordance with the [draft-ietf-httpapi-ratelimit-headers-07](https://datatracker.ietf.org/doc/draft-ietf-httpapi-ratelimit-headers/) spec but the ability to rename the headers already exists. Maybe a comment in the docs and a warning-level log alerting users to this fact might suffice?